public/assets/css/index.css

```diff
@@ -0,0 +1,64 @@
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
+}
+
+section {
+    width: 100%;
+}
+
+h1 {
+    color: rgba(219, 100, 100, 1);
+    -webkit-text-stroke: 2px rgba(121, 10, 77, 1);
+}
+
+.imgMedium {
+    max-width: 20%;
+    margin: 30px 0;
+}
+
+.versus {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+    margin: 30px 0;
+}
+
+.versus img {
+    width: 15%;
+    padding: 0 20px;
+}
+
+.versus .imgSmall {
+    width: 5%;
+}
+
+
+p {
+    color: rgba(219, 100, 100, 1);
```

Comment:
Could you use variable, in order to optimize color change and use. you can create a color variable in css (like "primary-color) and use it everywhere as soon as the variable is declared in the global css file